dom0/sd-logging-setup.sls

```diff
@@ -62,41 +62,33 @@ sd-gpg-remove-rsyslog-qubes-plugin:
     - require:
       - file: sd-gpg-remove-rsyslog-qubes-plugin
 
-{% elif grains['id'] == "sd-whonix" %}
-# We can not place the file on the template under /etc/rsyslog.d/ because of whonix
-# template. This sdlog.conf file is the same from the securedrop-log package, to
-# make sure that rsyslogd use our logging plugin.
-sd-rsyslog-sdlog-conf-for-sd-whonix:
-  file.managed:
-    - name: /rw/config/sdlog.conf
-    - source: "salt://sdlog.conf"
-
-# Because whonix-gw-15 template is not allowing to create the config file on
-# package install time, we do it via rc.local call.
```

Comment:
For the record, this was added in 232c56f44a34d393b89b0578364e955da62f1576. @kushaldas, can you elaborate on what you meant at the time with "whonix-gw-15 template is not allowing to create the config file on package install time"? As far as I can tell, the config file `/etc/rsyslog.d/sdlog.conf` is installed just fine from the package without it, and I've not observed any behavior specific to `whonix-gw-15` that would prevent this from being the case.

Reply:
https://github.com/freedomofpress/securedrop-workstation/pull/447/ this is the PR and as I can see in my logs, the file was still missing even after the package was installed.

Here is my note from that work:

```
* https://github.com/freedomofpress/securedrop-workstation/pull/447  howt to remove the files?
  - how to remove gpg key? sudo apt-key del KEY_ID
  - `make clean`
  - remove works properly 
  - `make all` agagin.
  - most of the vms decided not to reply to Qubes calls :(
  - cleaned, and `make all` again 
  - Someday Qubes and Salt will play nicely together. Question is in which year?
  - this try, the package is there, but, no logs. 
  - because the sdlog.conf file is missing, WHY WHY WHY
  - I this before, i commented on the issue, dpkg did not create the sdlog.conf file at all on the system.
  - one idea, if the file is missing, just copy it over via rc.local file
  - testing again 
  - testing again, adding sdlog.conf only for sd-whonix if missing.
  - testing once more, things look better now
  - meeting with Mickael
  - need to test some more changes
  - make clean and make all again at super late night :)
```

Reply:
Hi @kushaldas, I experienced this once during testing with `dpkg`, and in my experience it happens when you've previously removed the file by hand and uninstalled (rather than purged) the package - regardless of the order of the two operations. It's not specific to Whonix, and is just how Debian handles configuration files. Because we purge the file correctly with our uninstall logic, I don't believe it should be an issue on production systems.